platform/security/platform-security-core-api/src/main/java/ddf/security/SubjectUtils.java

```diff
@@ -161,36 +163,44 @@ public static String filterDN(X500Principal principal, Predicate<RDN> predicate)
     }
 
     public static String getEmailAddress(Subject subject) {
-        if (subject == null) {
-            LOGGER.debug("Incoming subject was null, cannot look up email address.");
+        List<String> values = getAttribute(subject, EMAIL_ADDRESS_CLAIM_URI);
+
+        if (values.isEmpty()) {
```

Comment:
Mind adding `@Nullable` on the return type and a comment explaining when this can happen?